libr/lang/p/lib.c

```diff
@@ -5,7 +5,7 @@
 static bool lang_lib_file_run(RLangSession *user, const char *file) {
 	char *libpath;
```

Comment:
but imho this should be an `R_RETURN_VAL_IF_FAIL (user && file, false);` not sure if it makes sense to call this api with nulls

```suggestion
	char *libpath = R_STR_DUP (file);
```